src/Interactive/Features/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class Service : MarshalByRefObject, IDisposable
         {
             private static readonly ManualResetEventSlim s_clientExited = new ManualResetEventSlim(false);
 
-            private static TaskScheduler s_UIThreadScheduler;
+            private static Control s_control;
```

Comment:
Your offline explanation sounded reasonable, but I have limited expertise in this area.